### PR TITLE
Add support for additional chocolatey package flags

### LINKED ISSUE #21: Add support for chocolatey flags
Add support for:

* `--prerelease` (i like to be bold on some packages)
* `--ignore-checksum` (the chrome package is notorious for a bad checksun)
* `--package-parameters` (some packages like the visual studio & sql server packages require some extra parameters)

### PR DESCRIPTION
**Pull request by BlythMeister**
fixes #21

### CODE REVIEW

File: src/Cupboard/Resources/Windows/Chocolatey/ChocolateyPackage.cs [hunk 1]
```diff
@@ -6,11 +6,15 @@ public sealed class ChocolateyPackage : Resource
     {
         public string Package { get; set; }
         public PackageState Ensure { get; set; }
+        public bool PreRelease { get; set; }
+        public bool IgnoreChecksum { get; set; }
+        public string PackageParameters { get; set; }
```
Comment (patriksvensson) [inline src/Cupboard/Resources/Windows/Chocolatey/ChocolateyPackage.cs:11]: Could we make this nullable?

**Review by patriksvensson — CHANGES_REQUESTED** [review]
LGTM! There's just a small thing I think needs to be changed, to make things more consistent.

File: src/Cupboard/Resources/Windows/Chocolatey/ChocolateyPackage.cs (hunk 1 above)
Comment (BlythMeister) [reply src/Cupboard/Resources/Windows/Chocolatey/ChocolateyPackage.cs:11]: the string or all 3?

File: src/Cupboard/Resources/Windows/Chocolatey/ChocolateyPackage.cs [hunk 2]
```diff
@@ -6,6 +6,9 @@ public sealed class ChocolateyPackage : Resource
     {
         public string Package { get; set; }
         public PackageState Ensure { get; set; }
+        public bool? PreRelease { get; set; }
+        public bool? IgnoreChecksum { get; set; }
```
Comment (patriksvensson) [inline src/Cupboard/Resources/Windows/Chocolatey/ChocolateyPackage.cs:10]: The booleans do not have to be nullable here.

**Review by patriksvensson — CHANGES_REQUESTED** [review]

File: src/Cupboard/Resources/Windows/Chocolatey/ChocolateyPackage.cs (hunk 2 above)
Comment (BlythMeister) [reply src/Cupboard/Resources/Windows/Chocolatey/ChocolateyPackage.cs:10]: 🤦‍♂️ I erred on the side of caution 🤣
I'll fix tomorrow.

File: src/Cupboard/Resources/Windows/Chocolatey/ChocolateyPackage.cs (hunk 1 above)
Comment (patriksvensson) [reply src/Cupboard/Resources/Windows/Chocolatey/ChocolateyPackage.cs]: Sorry, I missed this comment. I meant `PackageParameters`.

**Comment by BlythMeister** [comment]
@patriksvensson requested changes made :)

**Review by patriksvensson — APPROVED** [review]

**Comment by patriksvensson** [comment]
Merged! Thank you for your contribution. Much appreciated! 👍

**Pull request merged**